flink-yarn/pom.xml

```diff
@@ -113,6 +113,10 @@ under the License.
 						<jvmArg>-Xms128m</jvmArg>
 						<jvmArg>-Xmx512m</jvmArg>
 					</jvmArgs>
+					<args>
+						<arg>-Xmax-classfile-name</arg>
+						<arg>128</arg>
```

Comment:
According to the [man pages for EncFS](http://manpages.ubuntu.com/manpages/trusty/man1/encfs.1.html):

>  One such limitation is filename length.  If your underlying filesystem
       limits you to N characters in a filename, then EncFS will limit you to
       approximately 3*(N-2)/4.  For example if the host filesystem limits to
       256 characters, then EncFS will be limited to 190 character filenames.
       This is because encrypted filenames are always longer then plaintext
       filenames.

So that would be <190 characters for ext4, but it depends on the filesystem. 

I get the feeling that this creates unneeded headaches. I could change this PR into a docs one instead, and add a mention of the workaround in the docs as you suggested.